assets/_sass/_navbar.scss

```diff
@@ -91,7 +91,7 @@ $nav-item-border: darken($navbar-default-bg, 8%);
 
     .menu-row {
       background-color: transparent;
-      padding: 0 5px;
+      // padding: 0 5px;
```

Comment:
remove?

Reply:
Not sure this is still relevant? I think you might have had to refresh? Just double check because I cannot find this anywhere